net/third_party/quiche/src/quiche/common/platform/api/quiche_file_utils_test.cc

```diff
@@ -7,65 +7,137 @@
 #include "absl/strings/strip.h"
 #include "absl/types/optional.h"
 #include "quiche/common/platform/api/quiche_test.h"
+#if defined(STARBOARD)
+#include "base/files/file.h"
+#include "base/files/file_path.h"
+#include "base/files/file_util.h"
+#include "starboard/system.h"
+#endif
 
 namespace quiche {
 namespace test {
 namespace {
 
+#if defined(STARBOARD)
+base::FilePath GenerateTestDataInCache() {
```

Comment:
How are these files created upstream? Or should we be checking somewhere other than the cache for these files?

Reply:
Okay, but here we're manually writing these files and it doesn't seem like quiche itself needs to. Why do we need to manually create them and what does quiche do normally?